scripts/us_census/pep_srh/process.py

```diff
@@ -0,0 +1,1057 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+This module creates CSV files used for importing data into DC.
+
+Below are list of files processed -
+County
+    1990 - 2000     Processed As Is
+    2000 - 2010     Processed As Is
+    2010 - 2020     Processed As Is
+
+State
+    1980 - 1990     Processed As Is
+    1990 - 2000     Processed As Is
+    2000 - 2010     Aggregated from County to state
+                    (data matches with State Level files available)
+    2010 - 2020     Aggregated from County to state
+                    (data matches with State Level files available)
+
+National
+    1980 - 1990     Aggregted from State to National Level
+    1990 - 2000     Aggregted from State to National Level
+    2000 - 2010     Aggregted from State to National Level
+    2010 - 2020     Aggregted from State to National Level
+
+Also SV aggregation are produced while processing above files.
+E.g., Count_Person_White_HispanicOrLatino is calulated by addding
+Count_Person_Male_WhiteHispanicOrLatino and
+Count_Person_Male_WhiteHispanicOrLatino
+
+Before running this module, run download.sh script, it downloads required
+input files, created necessary folders for processing.
+
+Folder information
+download_files - downloaded files are placed here
+process_files - intermediate processed files are placed in this folder.
+output_files - output files (mcf, tmcf and csv are written here)
+'''
+
+import os
+import pandas as pd
+
+from constants import DOWNLOAD_DIR, PROCESS_AS_IS_DIR, PROCESS_AGG_DIR
+from constants import OUTPUT_DIR, stat_var_col_mapping, working_directories
+
+_CODEDIR = os.path.dirname(os.path.realpath(__file__))
+
+
+def process_national_files_1980_1990():
```

Comment:
Pls use function arguments to all functions below for input/output directories so that same function can be called in test with a different file and local output path.

Reply:
I have refactored code, as all national level files used similar block. Above review comment is taken care along.